aurora/scripts/utils.js

```diff
@@ -168,6 +172,45 @@ async function forceIncreaseBalance(signer, config, fastBridgeAddress, token, re
   console.log("Transaction hash: ", receipt.hash);
 }
 
+async function get_pending_transfers(config, nonce) {
```

Comment:
```suggestion
async function get_pending_transfer(config, nonce) {
```

This function returns just one transfer by nonce